tests/console/zypper_extend.pm

```diff
@@ -174,6 +176,8 @@ sub run {
     #Enter zypper shell and run the lr command | echo lr
     assert_script_run('echo lr |zypper shell');
 
+    assert_script_run('gpgconf --kill gpg-agent');
+    assert_script_run('rm -rf ~/.gnupg/*');
```

Comment:
That does not seem like a good idea, there might be important stuff in there (in theory).
It should probably be `/root/.gnupg/*` anyway

Reply:
> I seems like a product bug to me that:
> 
>     * zypper does stuff in `/root/.gnupg`
> 
>     * A running gpg-agent fails to have pinentry make proper use of the tty
> 
> 
> So I'd prefer it if this is detected as such and made an explicit softfail instead of always killing and deleting stuff as workaround.
> 
> Though this certainly needs some investigation first...

```
 rpm -q --scripts /var/cache/zypp/packages/packman/Essentials/noarch/rpmkey-packman-1.0.0-7.8.noarch.rpm        
warning: /var/cache/zypp/packages/packman/Essentials/noarch/rpmkey-packman-1.0.0-7.8.noarch.rpm: Header V4 RSA/SHA1 Signature, key ID 1abd1afb: NOKEY
postinstall scriptlet (using /bin/sh):
if [ ! -f usr/lib/rpm/gnupg/pubring.gpg ]; then   
.
.
.
### import suse package build key to roots gpg keyring
if test -f root/.gnupg/pubring.gpg ; then
   chroot . usr/bin/gpg --export --armor --no-default-keyring \
                   --keyring usr/lib/rpm/gnupg/packman.gpg packman@links2linux.de \
        | chroot . usr/bin/gpg --import || true
   if ! chroot . usr/bin/gpg --list-keys packman@links2linux.de >/dev/null 2>&1 ; then
      echo "gpg import for packman@links2linux.de failed, please import manually" >&2
   fi
else
   cp usr/lib/rpm/gnupg/packman.gpg root/.gnupg/pubring.gpg
fi
chmod 600 root/.gnupg/pubring.gpg
```

`rpmkey-packman` seems to cause all the troubles

Reply:
That's pure evil. I don't know where to report packman issues, but that definitely must be dropped.

Meanwhile, can the test be changed to work without that package?

Reply:
Sure xD, let me find some other replacement.